pyproject.toml

```diff
@@ -1,10 +1,81 @@
 [build-system]
-    build-backend = "setuptools.build_meta"
-    requires = [
-        "setuptools>=47",
-        "wheel>=0.34",
-        "cython>=3.0.8; python_implementation == 'CPython'", # Skip cython when using pypy
-    ]
+build-backend = "setuptools.build_meta"
+requires = [
+    "setuptools>=47",
+    "wheel>=0.34",
+    "cython>=3.0.8; python_implementation == 'CPython'", # Skip cython when using pypy
+]
+
+[project]
+name = "falcon"
+readme = "README.rst"
```

Comment:
Do we need to indicate that this is rst?

Reply:
as you prefer. a case may be made that being explicit is nicer, but it's fine as is since it's the default

Reply:
I think it is fine as it is because the most official example I could find uses exactly this form. OTOH, it doesn't hurt to be explicit, I can change.